shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -0,0 +1,234 @@
+      if (usedBytesOnTiers != null) {
+        for (Map.Entry<String, Long> usedBytesTier: usedBytesOnTiers.entrySet()) {
+          String tier = usedBytesTier.getKey();
+          long value = usedBytesTier.getValue();
+          sumUsedBytesOnTiersMap.put(tier, value + sumUsedBytesOnTiersMap.get(tier));
+        }
+      }
+    }
+
+    // Print summarized worker capacity information
+    if (options.getWorkerRange().equals(WorkerRange.SPECIFIED)
+        && sumCapacityBytes + sumUsedBytes == 0) {
+      System.out.println(USAGE);
+      throw new InvalidArgumentException("Worker IP addresses are invalid.");
+    }
+
+    mIndentationLevel = 0;
+    print(String.format("Capacity Information for %s Workers: ", options.getWorkerRange()));
+
+    mIndentationLevel++;
+    print("Total Capacity: " + FormatUtils.getSizeFromBytes(sumCapacityBytes));
+    mIndentationLevel++;
+    for (Map.Entry<String, Long> totalBytesTier : sumTotalBytesOnTiersMap.entrySet()) {
+      long value = totalBytesTier.getValue();
+      if (value != 0) {
+        print("Tier: " + totalBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      }
+    }
+
+    mIndentationLevel--;
+    print("Used Capacity: "
+        + FormatUtils.getSizeFromBytes(sumUsedBytes));
+    mIndentationLevel++;
+    for (Map.Entry<String, Long> usedBytesTier : sumUsedBytesOnTiersMap.entrySet()) {
+      long value = usedBytesTier.getValue();
+      if (value != 0) {
+        print("Tier: " + usedBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      }
+    }
+
+    mIndentationLevel--;
+    if (sumCapacityBytes != 0) {
+      int usedPercentage = (int) (100L * sumUsedBytes / sumCapacityBytes);
+      print(String.format("Used Percentage: " + "%s%%", usedPercentage));
+      print(String.format("Free Percentage: " + "%s%%", 100 - usedPercentage));
+    }
+    print("");
+
+    // Print cached worker information
+    mPrintStream.println(mStringBuilder.toString());
+  }
+
+  /**
+   * Gets the report worker options.
+   *
+   * @param cl CommandLine that contains the client options
+   * @return ReportWorkerOptions to get report worker information
+   */
+  private ReportWorkerOptions getOptions(CommandLine cl) throws IOException {
+    ReportWorkerOptions options = ReportWorkerOptions.defaults();
+    options.setFieldRange(new HashSet<>(Arrays.asList(ReportWorkerInfoField.ADDRESS,
+        ReportWorkerInfoField.CAPACITY_BYTES, ReportWorkerInfoField.CAPACITY_BYTES_ON_TIERS,
+        ReportWorkerInfoField.ID, ReportWorkerInfoField.LAST_CONTACT_SEC,
+        ReportWorkerInfoField.START_TIME_MS, ReportWorkerInfoField.USED_BYTES,
+        ReportWorkerInfoField.USED_BYTES_ON_TIERS)));
+    if (cl.hasOption("live")) {
+      options.setWorkerRange(WorkerRange.LIVE);
+    } else if (cl.hasOption("lost")) {
+      options.setWorkerRange(WorkerRange.LOST);
+    } else if (cl.hasOption("worker")) {
```

Comment:
this should be plural since there could be multiple

Reply:
Check if only one option has been passed in.

```java
private ReportWorkerOptions getOptions(CommandLine cl) throws IOException {
    if (cl.getOptions().length > 1) {
      System.out.println(getUsage());
      throw new InvalidArgumentException("Too many arguments passed in.");
    }
```